e2e/e2e-values.yml

```diff
@@ -18,6 +17,27 @@ resources:
   requests:
     cpu: 100m
     memory: 150M
-apiServerEndpointUrl: https://localhost:8443
-schedulerEndpointUrl: https://localhost:10259
-controllerManagerEndpointUrl: https://localhost:10257
+
+scrapersConfig:
+  interval: 15s
+  verbose: true
+  timeout: 30s
+  networkRouteFile: "/proc/net/route"
+
+  kubelet:
+    enabled: true
+
+  ksm:
+    enabled: true
+
+  controlPlane:
+    enabled: true
```

Comment:
should we keep the enabled config in true here? i think we are deploying this into 2 nodes en minikube and one of this is not the master

Reply:
anyway it should not fail :) so perhaps is good to test that

Reply:
there are node selectors, therefore the cp scraper is not deployed to the not-master nodes